docs/tutorials/quickstart-app-auth.md

```diff
@@ -153,12 +361,17 @@ const app = createApp({
 });
 ```
 
-7. Start the backend and frontend as before
+After finishing setting up one (or multiple) authentication providers defined
+above you can start the backend and frontend as before
 
 When the browser loads, you should be presented with a login page for GitHub.
 Login as usual with your GitHub account. If this is your first time, you will be
 asked to authorize and then are redirected to the catalog page if all is well.
 
+For more information you can clone the repository:
+https://github.com/RoadieHQ/backstage-auth-example Each authentication setting
```

Comment:
@core-team we should move this under the backstage GitHub org if you think it is useful.